From 135a74ae1814a1ae310d2f1faa19efe06bbfa964 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Wed, 29 Sep 2021 12:33:35 +0100
Subject: [PATCH] dt-bindings: media: i2c: Add binding for ad5398 VCM

Add a binding for Analog Devices AD5398 10bit current
sinking DAC when used as a lens VCM driver.

FIXME: Convert to YAML

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 .../devicetree/bindings/media/i2c/ad5398.txt  | 20 +++++++++++++++++++
 1 file changed, 20 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/media/i2c/ad5398.txt

--- /dev/null
+++ b/Documentation/devicetree/bindings/media/i2c/ad5398.txt
@@ -0,0 +1,20 @@
+* Analog Devices AD5398 autofocus coil
+
+Required Properties:
+
+  - compatible: Must contain one of:
+		- "adi,ad5398"
+
+  - reg: I2C slave address
+
+  - VANA-supply: supply of voltage for VANA pin
+
+Example:
+
+       ad5398: coil@c {
+               compatible = "adi,ad5398";
+               reg = <0x0c>;
+
+               VANA-supply = <&vaux4>;
+       };
+
